src/it/scala/net/snowflake/spark/snowflake/SnowflakeResultSetRDDSuite.scala

```diff
@@ -995,6 +995,7 @@ class SnowflakeResultSetRDDSuite extends IntegrationSuiteBase {
       )""".stripMargin
     )
     awsConn.createStatement.execute("GET @test_move_data/ file:///tmp/test_move_data")
+    awsConn.close()
```

Comment:
I assume this change (and the other changes like this one) are not related to telemetry?

Reply:
Correct!
I browsed all connections built in spark connector, I found some connections are not closed yet. So, I close them.